static/js/main.js

```diff
@@ -127,6 +128,45 @@ function goBack(){
     table.innerHTML = tabletext
 }
 
+
+function getNodes(url){
+    var jsonobj = ''
+    var xhttp = ''
+    try{
+        xhttp = new XMLHttpRequest()
+    } catch(e){
```

Comment:
This whole try/catch isn't needed. XMLHttpRequest has been supported in all browsers since IE7 which was released in 2006. Drop the try/catch and just keep the initialization.

Reply:
Aight, I just thought it was funny.  I'll take it out.